jpa/src/main/java/org/jboss/as/jpa/container/AbstractEntityManager.java

```diff
@@ -62,8 +62,6 @@
     public static final String PESSIMISTIC_WRITE_LOCK_MODE = "pessimistic_write";
     public static final String NONE_LOCK_MODE = "none";
 
-    private final transient boolean isTraceEnabled = ROOT_LOGGER.isTraceEnabled();
```

Comment:
Right; the commit that does this part can just rebased out, or the change reverted in some other fashion.

Trace logging config changes will still be picked up by newly instantiated objects, and in most cases the lifespan of these objects is short. Even in the SFSB with XPC case it will probably be fine in practice, as new SFSBs will be instantiated frequently enough to get useful trace logging.